pages/linux/setfont.md

```diff
@@ -0,0 +1,20 @@
+# setfont
+
+> Set the terminal font.
+> More information: <https://manned.org/setfont>.
+
+- Change the terminal font:
+
+`setfont {{font}}.gz`
+
+- Change the terminal font by specifying the full filepath:
+
+`setfont /usr/share/kbd/{{consolefonts|unimaps|consoletrans}}/{{font}}.gz`
```

Comment:
Im unaware, is it exclusively these or can it be other directories (consolefonts|unimaps etc)

Reply:
Might it be better to just have `/usr/share/kbd/{{mappings}}/{{font}}.gz just for absolutely certainty.

Reply:
There's not that many directories in /usr/share/kbd so I'll take that.